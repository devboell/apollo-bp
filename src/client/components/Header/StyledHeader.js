import styled from 'styled-components';

const StyledHeader = styled.header`
  display: flex;
  background-color: lightblue;
`;

export default StyledHeader;
