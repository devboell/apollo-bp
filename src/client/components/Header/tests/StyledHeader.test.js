import React from 'react'
import 'jest-styled-components'

import StyledHeader from '../StyledHeader'

it('StyledHeader, styled component snapshot', () => {
  const wrapper = shallow(<StyledHeader />)
  expect(wrapper).toMatchSnapshot()
})
