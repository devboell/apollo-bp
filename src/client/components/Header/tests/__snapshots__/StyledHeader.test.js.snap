// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StyledHeader, styled component snapshot 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  vertical-align: center;
  padding: 10px 0px 20px 30px;
}

<header
  className="c0"
/>
`;
